README.md

```diff
@@ -28,10 +28,17 @@ solcjs --help
 
 To compile a contract that imports other contracts via relative paths:
 ```bash
-solcjs --bin --base-path . ./MainContract.sol
+solcjs --bin --include-path node_modules/ --base-path . MainContract.sol
 ```
-The option ``--base-path`` enables automatic loading of imports from the filesystem and
-takes a path as argument that contains the source files.
+Use the ``--base-path`` and `--include-path`` options to describe the layout of our project.
```

Comment:
```suggestion
Use the ``--base-path`` and ``--include-path`` options to describe the layout of your project.
```

Reply:
Haha, interesting that `our` is just a typo away from `your` and results in something that still kinda makes sense :)